use-cases-working-draft.md

```diff
@@ -296,7 +296,7 @@ The requirements for Interchain NFTs include that these should be:
 5. **Non-transferable** to not permit changes in ownership.
 6. **Lockable** to conditionally stop changes in the token metadata.
 7. **Immutable** to never permit changes in the token metadata.
-8. **Mutable** to unconditionally permit changes in the token metadata.
+8. **Mutable** to conditionally permit changes in the token metadata.
```

Comment:
@okwme if you could please revert your change from `unconditionally` to `conditionally`, we can merge the rest of your changes, thanks

Reply:
```suggestion
8. **Mutable** to unconditionally permit changes in the token metadata.
```

Reply:
👍